.github/workflows/_check-production-node-and-runtime.yml

```diff
@@ -0,0 +1,30 @@
+---
+# This workflow checks production version of aleph-node and aleph-runtime, ie
+# only compilation and not linking, which takes most of the time
+name: Check production node and runtime
+on:
+  workflow_call:
+
+jobs:
+  main:
+    name: Build production node and runtime
+    runs-on: [self-hosted, Linux, X64, large]
+    env:
+      RUST_BACKTRACE: full
+      RUSTC_WRAPPER: sccache
+    steps:
+      - name: Checkout aleph-node source code
+        uses: actions/checkout@v4
+
+      - name: Call action get-ref-properties
+        id: get-ref-properties
+        # yamllint disable-line rule:line-length
+        uses: Cardinal-Cryptography/github-actions/get-ref-properties@v2
+
+      - name: Install Rust toolchain
+        uses: Cardinal-Cryptography/github-actions/install-rust-toolchain@v2
+        with:
+          targets: wasm32-unknown-unknown
+
+      - name: Build production binary and runtime
+        run: cargo check --profile production -p aleph-node
```

Comment:
Hm, all the other files seem to be using `actions-rs/cargo`, while here the command is explicit, is this on purpose?

Reply:
I think it's the other way round, most of examples use bare commands as we use our action to install rust toolchain. We use actions-rs/cargo when invoking `cargo test` ie when `wasm32-unknown-unknown` is not needed. Good point though, we will unify those rust build actions in further PR.